README.md

```diff
@@ -1,87 +1,79 @@
 # CSI NFS driver
 
-## Kubernetes
-### Requirements
+## Overview
 
-The folllowing feature gates and runtime config have to be enabled to deploy the driver
+This is a repository for [NFS](https://en.wikipedia.org/wiki/Network_File_System) [CSI](https://kubernetes-csi.github.io/docs/) Driver.
+Currently it implements bare minimum of the [CSI spec](https://github.com/container-storage-interface/spec) and is in the alpha state 
+of the development.
 
-```
-FEATURE_GATES=CSIPersistentVolume=true,MountPropagation=true
-RUNTIME_CONFIG="storage.k8s.io/v1alpha1=true"
-```
+#### CSI Feature matrix
 
-Mountprogpation requries support for privileged containers. So, make sure privileged containers are enabled in the cluster.
+| **nfs.csi.k8s.io** | K8s version compatibility | CSI versions compatibility | Dynamic Provisioning | Resize | Snapshots | Raw Block | AccessModes              | Status                                                                       |
+|--------------------|---------------------------|----------------------------|----------------------|--------|-----------|-----------|--------------------------|------------------------------------------------------------------------------|
+|master              | 1.14 +                    | v1.0 +                     |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha                                                                        |
+|v2.0.0              | 1.14 +                    | v1.0 +                     |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | Alpha                                                                        |
+|v1.0.0              | 1.9 - 1.15                | v1.0                       |  no                  |  no    |  no       |  no       | Read/Write Multiple Pods | [deprecated](https://github.com/kubernetes-csi/drivers/tree/master/pkg/nfs)  |
```

Comment:
why is the max 1.15?

Reply:
https://github.com/kubernetes-csi/drivers/blob/337ec802362eb64fa9b6abd1a1b14ad997575d72/pkg/nfs/deploy/kubernetes/csi-nodeplugin-nfsplugin.yaml#L4

uses API group `apps` deprecated in 1.16